### PR TITLE
set table responsive for media 760px and below

### PR DESCRIPTION
**Pull request by Amutha37**
The css for table responsive is added. It currently responsive.

### CODE REVIEW

File: pages/form/Question2.js [hunk 1]
```diff
@@ -1,69 +1,79 @@
...
-              </Select>
-            </Box>
-            <Grid templateColumns="repeat(2, 1fr)" gap={4}>
-              <BackButton href="/form/Question1" onClick={saveAnswers} />
-              <ContinueButton href="/form/Question3" onClick={saveAnswers} disabled={!selectedMode} />
-            </Grid>
-          </>
-      }
+      <Q2Progress  />
+      {isWFH ? (
```
Comment (hqtan) [inline pages/form/Question2.js:27]: can we use an `if ... else...` block here instead?
```suggestion
      if(isWFH) {
```

File: pages/form/Question2.js [hunk 2]
```diff
@@ -1,69 +1,79 @@
 import { useState } from "react";
-import {
-  Box,
-  Grid,
-  Heading,
-  Text,
-  Select,
-} from "@chakra-ui/react";
+import { Box, Grid, Heading, Text, Select } from "@chakra-ui/react";
```
Comment (hqtan) [inline pages/form/Question2.js:2]: let's be consistent with our import statements, and keep each imported component on a separate line k?
```suggestion
import {
  Box,
  Grid,
  Heading,
  Text,
  Select,
} from "@chakra-ui/react";
```

File: pages/form/Question1.js [hunk 3]
```diff
@@ -39,33 +38,55 @@ export default function Question1() {
...
-    }))
-  }
+    }));
+  };
 
   return (
     <Layout>
-      <Q1Progress /> 
+      <Q1Progress className="responsivebar" />
+      {/* <Q1Progress className="responsivebar" viewBox="100 5 490 50" height="50" width="100%" /> */}
```
Comment (hqtan) [inline pages/form/Question1.js:53]: do we need this comment?
```suggestion
```

**Review by hqtan — CHANGES_REQUESTED** [review]
Thanks for your PR @Amutha37 !

2 things I *really* liked with this PR:
- the CSS work you did is awesome! the table in Question 1 looks **much** nicer on a small screen with those css settings! :heart_eyes: 
- the tidying up of the code, in general :raised_hands: ... although i'm still not sure how I feel about the proliferation of semicolons in some places :sweat_smile: 

I've left some comments about some things to tidy up in this PR (see below), and also request that you please remove `package-lock.json`; it seems you have used npm v5 or v6 to regenerate the lockfile (judging by the [`lockfileVersion number in the file`](https://docs.npmjs.com/cli/v7/configuring-npm/package-lock-json#lockfileversion) . Please use npm v7 instead.

**Comment by hqtan** [comment]
Hey @batteries76 , @ymloh ,
I'd appreciate your feedback on this PR, in case I've missed something...

**really** happy how this PR fixes how `form/Question1` renders on small screen :+1: ; what do you think?

File: styles/globals.css [hunk 4]
```diff
@@ -14,3 +15,83 @@ a {
...
+*/
+@media only screen and (max-width: 760px),
+  (min-device-width: 768px) and (max-device-width: 1024px) {
+  .square-grid {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+  }
+
+  .responsivebar {
```
Comment (hqtan) [inline styles/globals.css:34]: I think the css settings for `responsivebar` still needs some work @Amutha37...

this is a screenshot of the progress bar in see from my mobile device (Pixel4a) - see how the progress bar isn't fitting in screen?
image

**Review by hqtan — CHANGES_REQUESTED** [review]

**Comment by hqtan** [comment]
[[WIP] Dev: Form must be responsive](https://trello.com/c/QtxdOlzq/200-wip-dev-form-must-be-responsive)

**Comment by ymloh** [comment]
@Amutha37 @hqtan @batteries76 

I really like the idea of stacking the day on top of the checkboxes too! 🙌 

A couple of issues I'm aware of (doesn't necessarily need addressing):
- The original problem of the text wrapping in the checkbox labels persists.
- (subjective) there is a faint line bordering the days and the checkboxes on each table row.
- (subjective) I feel we should stick to the CSS library as far as we can before resorting to custom CSS, primarily in the name of maintainability.

What does everyone else think about these points?

File: styles/globals.css [hunk 5]
```diff
@@ -14,3 +15,83 @@ a {
...
+  html {
+    font-size: calc(16px + (24 - 16) * (100vw - 400px) / (800 - 400));
+  }
+}
+
+@media screen and (min-width: 50em) {
+  html {
+    font-size: calc(16px + (24 - 16) * (100vw - 400px) / (800 - 400));
+  }
+} */
```
[the remark this answers is not in the document]
Comment (Amutha37) [reply styles/globals.css:97]: Sorry I normally have a habit of securing old lines incase decide to use them. As I change preferences. It is deleted now.

File: styles/globals.css (hunk 4 above)
Comment (Amutha37) [reply styles/globals.css:34]: Yes, did try to work on the svg bar. It's acting funny I will try to work on it. Anyone else want to have a go on the svg bar try it too.

File: pages/form/Question1.js (hunk 3 above)
Comment (Amutha37) [reply pages/form/Question1.js:53]: This is the part you are mentioning need some work. I tried working on svg file. You know it an image pre fix with size. The image on the bar is what I'm struggling to get them responsive. If anyone can explain how I can do that?

File: pages/form/Question2.js (hunk 2 above)
Comment (Amutha37) [reply pages/form/Question2.js:2]: I have remove the additional line of importing from chakra-ui.

File: pages/form/Question1.js (hunk 3 above)
Comment (Amutha37) [reply pages/form/Question1.js:53]: This is the part I'm trying to work on the svg bar and the image. I might need some help on the svg image on the bar if anyone familiar with making it responsive. The is a kind on seperate to normally how I have done with images to be responsive. I will still search up for tips.

**Comment by Amutha37** [comment]
Change requested are resolved in code files. Closing the pull request.

**Pull request closed without merging**